src/components/Stepper/MucStepperItem.vue

```diff
@@ -4,13 +4,13 @@
     @click="handleClick"
   >
     <div
-      class="m-form-step__icon"
+      :class="'m-form-step__icon' + disabledStep"
```

Comment:
```suggestion
      class="m-form-step__icon"
      :class="{disabled: disabled}"
```
Please use the vue build in way of toggling css classes [see here](https://vuejs.org/guide/essentials/class-and-style) instead of hardcoding them into the computed property.

Reply:
this code was done freehand so please check if it works correctly